### PR TITLE
Added redirect to correct repository link

### LINKED ISSUE #234: Clicking on github link on project redirects to github.com but it should redirect to project repository
### Describe the bug.

https://user-images.githubusercontent.com/87171452/145668768-c3b1f64e-ea0e-4643-91a4-17ca4cfbd52b.mp4

### Describe the steps to reproduce the behavior.

Go to https://community.moja.global/roadmap
Scroll down and click on view timeline 
Click on github 
See error 

### Expected behavior.

Github link should be redirected to Repository link 

### Screenshots.

_No response_

### Operating Environment

Windows 10 and Chrome Browser 

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Palaksharma23**
Fixes #234

# Pull Request Template

## Description
Initially when we click on github link it was redirecting us to www.github.com but i have added link to redirect to particular specific repository of the project.

## Type of change

Please delete options that are not relevant.

- [yes] Bug fix (non-breaking change which fixes an issue).

#### INITIALLY

Go to https://community.moja.global/roadmap
Scroll down and click on view timeline
Click on github
See error

## Additional Context (Please include any Screenshots/gifs if relevant)

https://user-images.githubusercontent.com/87171452/145672398-0877e790-60bf-4798-b0be-5d0a252a9f65.mp4

...

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by Palaksharma23** [comment]
@HarshCasper can you please once review my pull request

File: website/src/pages/ProjectCard/ProjectCard.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const ProjectCard = (props) => {
           </div>
         </div>
         <div className="modal_links">
-          <a href="https://github.com/" className="modal_gh_link">
+          <a href={props.link} className="modal_gh_link">
```
Comment (sohamsshah) [inline website/src/pages/ProjectCard/ProjectCard.js:43]: Can we have `target="_blank" rel="noopener noreferrer"`, so that the project opens in a new tab?

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Hey @Palaksharma23 , thanks for pointing this out. Looks good to me!

File: website/src/pages/ProjectCard/ProjectCard.js (hunk 1 above)
Comment (Palaksharma23) [reply website/src/pages/ProjectCard/ProjectCard.js:43]: Sure sir @sohamsshah I will commit the changes asap :)

**Comment by Palaksharma23** [comment]
Done @sohamsshah 

https://user-images.githubusercontent.com/87171452/145680478-17731941-9d31-4c4b-8afc-d1174bd5f108.mp4

**Comment by sohamsshah** [comment]
LGTM! Thanks for contributing! 🥳

**Pull request merged**